EXPRESSION_EVALUATION.md

```diff
@@ -0,0 +1,180 @@
+# Submissions and Expression Evaluation in Rawls
+
```

Comment:
Where should this document live?  Should it be linked from the README?  Should it reference `EXPRESSIONS.md` or should the information in that document be included in this one?